auslib/test/test_db.py

```diff
@@ -20,6 +21,12 @@
 from auslib.blobs.apprelease import ReleaseBlobV1
 
 
+def setUpModule():
+    # This is meant to silence the debug information coming from SQLAlchemy-Migrate since
+    # AUSDatabase provides decent debugging logs by itself.
+    logging.getLogger('migrate').setLevel(logging.CRITICAL)
```

Comment:
This is awesome - it really reduces the spamminess of failures. If you don't mind, could you add it to the other test files that set up databases, too?